### PR TITLE
fix mac build

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/Paddle/issues/11871

### CODE REVIEW

File: cmake/cblas.cmake [hunk 1]
```diff
@@ -83,20 +83,6 @@ else()
   set(REFERENCE_CBLAS_LIB_SEARCH_PATHS ${REFERENCE_CBLAS_ROOT}/lib)
 endif()
 
-find_path(REFERENCE_CBLAS_INCLUDE_DIR NAMES cblas.h PATHS
-        ${REFERENCE_CBLAS_INCLUDE_SEARCH_PATHS})
-find_library(REFERENCE_CBLAS_LIBRARY NAMES cblas PATHS
-        ${REFERENCE_CBLAS_LIB_SEARCH_PATHS})
-
-if(REFERENCE_CBLAS_INCLUDE_DIR AND REFERENCE_CBLAS_LIBRARY)
```
Comment (Superjomn) [inline cmake/cblas.cmake]: why here delete these lines?

Comment (typhoonzero) [reply cmake/cblas.cmake]: Added comments:
```
# TODO(typhoonzero): I deleted those REFERENCE_CBLAS_INCLUDE_DIR things
# that may cause builds fail if we installed other cblas libs like "veclib",
# in paddle/fluid/operators/math/blas.h which will throw "Not Implemented"
# for other cblas. Openblas and MKL are supported only.
```

Comment (luotao1) [reply cmake/cblas.cmake]: Now indeed only Openblas and MKL are supported. We don't test other cblas libs. Why should you install and use other cblas libs?

Comment (typhoonzero) [reply cmake/cblas.cmake]: The other software installed veclib as a dependency. So I deleted this

Comment (luotao1) [reply cmake/cblas.cmake]: Though other software installed veclib, they can use MKL/openblas as well.

Comment (typhoonzero) [reply cmake/cblas.cmake]: If we need to keep this for future adding libraries, can add a switch here to determine whether to do `find_library`.

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**